litmus-portal/frontend/src/pages/GetStartedPage/index.tsx

```diff
@@ -53,6 +54,31 @@ const GetStarted: React.FC = () => {
 
   const [loading, setIsLoading] = useState<boolean>(false);
 
+  // Checking if token is valid or not by finding the uid in database
+  const ValidateUser = () => {
+    fetch(`${config.auth.url}/getUser/${getUserId()}`, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${getToken()}`,
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        if ('error' in data) {
+          console.error('User not found');
```

Comment:
Can we display the error here also?

Reply:
We can display it, but if the UI encounters an error, then the cookie is cleared and page is refreshed and therefore everything from the console is cleared. It's quite difficult to even see that error